SmartDeviceLink/public/SDLVideoStreamingCapability.m

```diff
@@ -30,6 +30,7 @@ - (instancetype)initWithPreferredResolution:(nullable SDLImageResolution *)prefe
     self.diagonalScreenSize = @(diagonalScreenSize);
     self.pixelPerInch = @(pixelPerInch);
     self.scale = @(scale);
+    self.preferredFPS = @(15);
```

Comment:
This is incorrect, the initializer should be deprecated and replaced. Please generate the file using the RPC generator and the correct branch of the rpc_spec. You'll need to modify the existing file instead of replacing it with the new file, but it will ensure that the new pieces are created correctly.

Reply:
Because RPC generator did not exist when I did initially pull-requested (which was in March), SDLVideoStreamingCapability was manually edited as you pointed out. I have updated the code based on RPC generator's generated code. Thanks.